### PR TITLE
[14.0][MIG] partner phonecall schedule

### PR DESCRIPTION
**Pull request by redapureskill**
14.0 mig partner phonecall schedule

### CODE REVIEW

**Comment by redapureskill** [comment]
Approve !

**Comment by redapureskill** [comment]
Waiting Approval!

File: partner_phonecall_schedule/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_phonecall_schedule/__init__.py:1]: Please change all license lines to https:

**Review by NL66278 — APPROVED** [review]
:+1: One request, to change license lines to https:, but does look good to me.

**Comment by dreispt** [comment]
It looks like this was superseded by https://github.com/OCA/partner-contact/pull/1175

**Pull request closed without merging**